samples/notebooks/dotnet/Getting-Started-Notebook.ipynb

```diff
@@ -135,32 +105,22 @@
     },
     "polyglot_notebook": {
      "kernelName": "csharp"
-    },
-    "vscode": {
-     "languageId": "polyglot-notebook"
     }
    },
    "outputs": [],
    "source": [
     "// Configure AI backend used by the kernel\n",
-    "var (useAzureOpenAI,model, azureEndpoint, apiKey, orgId) = Settings.LoadFromFile();\n",
+    "var (useAzureOpenAI, model, azureEndpoint, apiKey, orgId) = Settings.LoadFromFile();\n",
     "if (useAzureOpenAI)\n",
-    "    kernel.Config.AddAzureOpenAICompletionBackend(\"davinci\", model, azureEndpoint, apiKey);\n",
+    "    kernel.Config.AddAzureOpenAITextCompletion(\"davinci\", model, azureEndpoint, apiKey);\n",
     "else\n",
-    "    kernel.Config.AddOpenAICompletionBackend(\"davinci\", model, apiKey, orgId);"
+    "    kernel.Config.AddOpenAITextCompletion(\"davinci\", model, apiKey, orgId);"
```

Comment:
![image](https://user-images.githubusercontent.com/2929102/228738482-0d23f4a6-5f1b-436f-805b-2d1f9a87f834.png)

In this situation, it seems that the error won’t be resolved. I believe that adding the following line might help us to call the method.

`"using Microsoft.SemanticKernel.Configuration;\n",`

see. https://github.com/microsoft/semantic-kernel/pull/227/files#diff-e0ebc1b70fd7adeb09b564654fe276ed1f141c9865c63d8fb86b499e1e3e8248R121

Reply:
I closed the following #226 #227 to integrate them into the PR over here